osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -32,14 +36,23 @@ public OsuPerformanceCalculator()
         protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo score, DifficultyAttributes attributes)
         {
             var osuAttributes = (OsuDifficultyAttributes)attributes;
+            useClassicSlider = score.Mods.Any(h => h is OsuModClassic cl && cl.NoSliderHeadAccuracy.Value);
 
             accuracy = score.Accuracy;
             scoreMaxCombo = score.MaxCombo;
             countGreat = score.Statistics.GetValueOrDefault(HitResult.Great);
             countOk = score.Statistics.GetValueOrDefault(HitResult.Ok);
             countMeh = score.Statistics.GetValueOrDefault(HitResult.Meh);
             countMiss = score.Statistics.GetValueOrDefault(HitResult.Miss);
-            effectiveMissCount = calculateEffectiveMissCount(osuAttributes);
+            countLargeTickMiss = score.Statistics.GetValueOrDefault(HitResult.LargeTickMiss);
+
+            if (!useClassicSlider)
+                countSliderEndsDropped = osuAttributes.SliderCount - score.Statistics.GetValueOrDefault(HitResult.SliderTailHit);
+
+            if (useClassicSlider)
```

Comment:
Do stable scores have `NoSliderHeadAccuracy` set?

Reply:
idk but the desired effects are achieved when using the perfcalc tools so it seems to be fine?

Reply:
If we are worried about this.
we can add `score.IsLegacyScore` to conditions to just in case.

```cs
usingClassicSliderMechanics = score.Mods.Any(h => h is OsuModClassic cl && cl.NoSliderHeadAccuracy.Value) || score.IsLegacyScore
```

Reply:
> If we are worried about this. we can add `score.IsLegacyScore` to conditions to just in case.
> 
> ```cs
> usingClassicSliderMechanics = score.Mods.Any(h => h is OsuModClassic cl && cl.NoSliderHeadAccuracy.Value) || score.IsLegacyScore
> ```

no need to worry about this
you can't have stable score without CL unless someone replay-edited it specifically

Reply:
just curious, is there any real reason to not have it?

Reply:
> just curious, is there any real reason to not have it? (just as an extra safety guard)
the potential reason is when you have some score simulator like Osu-Tools, and you don't wanna see additional `IsLegacy` checkboxes everywhere (you don't have to if everything is done correctly but still)

Reply:
resolving because #27063 is merged so I assume it's fine :)

if I'm wrong, just reopen this